lang/llvm-4.0/Portfile

```diff
@@ -117,27 +117,27 @@ if {${distfiles} ne ""} {
     }
 }
 
-checksums           llvm-4.0.0.src.tar.xz \
-                    rmd160  468ae2502ba523b35c5e8340724ddaa50b31ccb5 \
-                    sha256  8d10511df96e73b8ff9e7abbfb4d4d432edbdbe965f1f4f07afaf370b8a533be \
-                    cfe-4.0.0.src.tar.xz \
-                    rmd160  aae742d32a18cd8660b0eabecfcf14139f2f4d4a \
-                    sha256  cea5f88ebddb30e296ca89130c83b9d46c2d833685e2912303c828054c4dc98a \
-                    compiler-rt-4.0.0.src.tar.xz \
-                    rmd160  82af42752eb7cf9f90cc678263596b713e642067 \
-                    sha256  d3f25b23bef24c305137e6b44f7e81c51bbec764c119e01512a9bd2330be3115 \
-                    libcxx-4.0.0.src.tar.xz \
-                    rmd160  b1a870353076c9af77c16f182d5d6a987a0e3030 \
-                    sha256  4f4d33c4ad69bf9e360eebe6b29b7b19486948b1a41decf89d4adec12473cf96 \
-                    clang-tools-extra-4.0.0.src.tar.xz \
-                    rmd160  1fae1678b524ec44c3eaed9bbe8d1accd2c85ec0 \
-                    sha256  41b7d37eb128fd362ab3431be5244cf50325bb3bb153895735c5bacede647c99 \
-                    lldb-4.0.0.src.tar.xz \
-                    rmd160  a4f8d036901ae17ca0959dc4cd1064bbbbaa05af \
-                    sha256  2dbd8f05c662c1c9f11270fc9d0c63b419ddc988095e0ad107ed911cf882033d \
-                    polly-4.0.0.src.tar.xz \
-                    rmd160  5e37430f0bc80a9a1acfce1643c02815609b2504 \
-                    sha256  27a5dbf95e8aa9e0bbe3d6c5d1e83c92414d734357aa0d6c16020a65dc4dcd97
+checksums           llvm-4.0.1.src.tar.xz \
+                    rmd160  37387a5ca73ea270b2f541ecbd1cd641f7b09be2
```

Comment:
Backslashes are missing here.

Please don't submit PRs without doing even the most basic check. This change completely breaks MacPorts due to a parse error.

(A question to @jeremyhu: why are all the numbers hardcoded?)